src/components/Input/docs/Input.stories.mdx

```diff
@@ -0,0 +1,168 @@
+import { useState, ChangeEvent } from 'react';
+import { Canvas, Story, ArgsTable, Meta } from '@storybook/addon-docs';
+import { Input } from '../Input';
+import { ControlledInput } from './ControlledInput';
+import { StyledSystemLinks } from '../../../docs/StyledSystemLinks';
+import { Combination } from '../../../docs/Combination';
+import { InputPropsTable } from './InputPropsTable';
+
+<Meta
+    title="Components/Input"
+    component={Input}
+    argTypes={{
+        variant: {
+            options: ['boxed', 'bottom-lined'],
+            defaultValue: 'boxed',
+            control: 'select'
+        },
+        size: {
+            options: ['small', 'medium'],
+            defaultValue: 'medium',
+            control: 'select'
```

Comment:
I think it would be good idea to aim to cover all props in `argTypes` with this migration (inverted, placeholder, etc.) with relevant controls per each prop type, so that our components are fully interactive on the Canvas tab (especially as this becomes a single view in Storybook v7, while Docs tab will be moved to sidebar)

Reply:
They are derived automatically by the Storybook, aren't they?

Reply:
They may be derived for documentation purposes yes, but in order to interact with them in Canvas (which I think is probably the most important feature of SB), we need to declare their controls like here in `argTypes` (unless that changed in SB v7)